### DISCUSSION TITLE
Some new stuff

### DISCUSSION THREAD

**Discussion by madprops** [madprops/grasshopper#111] [Announcements]
Titles can be colored. There's shortcuts for 3 RGB colors.

![muscm234](https://github.com/madprops/grasshopper/assets/206498/077216d8-ad33-432d-91c2-ac0043782061)

![title2n34](https://github.com/madprops/grasshopper/assets/206498/ebeeedc3-e207-495c-8fc9-bc79b7cf26bc)

---

There's now a clock that appears as the placeholder of the filter as you can see.

![muscm234](https://github.com/madprops/grasshopper/assets/206498/a91aac7b-adea-47b8-87f0-3bb98f96a6c8)

This can be set to 12 hours or 24 hours.

![clock123](https://github.com/madprops/grasshopper/assets/206498/c7697632-1999-4cdf-b78a-1d0b584a686e)

This is set to 12 hours by default.

---

There's now 3 URLs that can be set in General.

You can map these to actions by using commands like `Open URL One`.

![urlsgh](https://github.com/madprops/grasshopper/assets/206498/ab1cd2cb-40ff-4f72-8ebf-c8a96b04dd3f)

This is fixed to 3 for now.

**Comment by N3C2L** [comment]
Great additions! Thanks!

**Comment by madprops** [comment]
Added the ability to show the date as the title.

![titledate](https://github.com/madprops/grasshopper/assets/206498/306d2672-4a85-4761-9ebc-c5ee2ff2c1fc)

Double clicking the Filter toggles date now.
